static/src/javascripts/projects/common/modules/commercial/track-rubicon-ad-resize.js

```diff
@@ -0,0 +1,60 @@
+define([
+    'common/modules/commercial/track-ad'
+], function (
+    trackAd
+) {
+    function trackRubiconAdResize(id) {
+        return trackAd.track(id)
+            .then(getIframeId)
+            .then(listenForRubicon)
+            .then(getDims);
+
+        function getIframeId(isLoaded) {
+            if (!isLoaded) {
+                return false;
```

Comment:
Currently this fn returns either `Boolean | String`. Would it be simpler if the fn returned `String | undefined`? In Scala this would be modelled as `Option[String]`.

``` js
if (isLoaded) { return iframe.id }
```